### DISCUSSION TITLE
PAG load using api response

### DISCUSSION THREAD

**Discussion by kchloe** [Tencent/libpag#521] [Q&A] [answered]
Hi, I have a question to use PAGView

I want to know how can I use PAGView with url response.
below figure is example code for web integration.
It seems like we can make a file from url response.
#### Is there android example code for same operation?
![image](https://user-images.githubusercontent.com/28251694/189013785-dd2be455-b68b-4969-bcd2-519012b06c6a.png)

**Comment by domchen** [comment, marked as answer]
Loading PAGFile from url is supported since PAG 4.3 on the Android platform. But it is currently a beta version.  You can download it here:https://github.com/Tencent/libpag/releases  You can pass the url into the LoadAsync() / Load() methods of PAGFile or PAGView to load the file via network now.
